sftp/Containerfile

```diff
@@ -0,0 +1,27 @@
+# Use Alpine Linux as the base image
+FROM alpine:latest
+
+# Install necessary packages
+RUN apk add --no-cache \
+	openssh-server \
+	openssh-sftp-server \
+	mariadb-client \
+	linux-pam \
+	openssh-server-pam \
+	bash
+
+# Create a user for SFTP access
+RUN addgroup -g 991 asterisk && \
+	adduser -D -u 990 -h /var/sftp -s /sbin/nologin -G asterisk asterisk && \
+	mkdir -p /var/sftp && \
+	chown root:root /var/sftp && \
+	chmod 755 /var/sftp
+
+# Copy configuration files
+COPY sshd_config /etc/ssh/sshd_config
+COPY pam_config /etc/pam.d/sshd
+COPY check_mysql_password.sh /usr/local/bin/check_mysql_password.sh
+COPY entrypoint.sh /entrypoint.sh
+
+ENTRYPOINT ["/entrypoint.sh"]
+CMD ["/usr/sbin/sshd.pam", "-D", "-e"]
```

Comment:
The command `/usr/bin/sshd.pam -D -e` is already defined as the last step of the ENTRYPOINT script. Also, the entry point script does not manage passed arguments, so the `CMD` will always be ignored.

https://docs.docker.com/reference/dockerfile/#understand-how-cmd-and-entrypoint-interact

```suggestion
```